WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazePaymentMethodsView.swift

```diff
@@ -0,0 +1,276 @@
+import SwiftUI
+
+struct BlazePaymentMethodsView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+    @ObservedObject private var viewModel: BlazePaymentMethodsViewModel
+    @Environment(\.dismiss) private var dismiss
+    @State private var showingAddPaymentWebView: Bool = false
+
+    init(viewModel: BlazePaymentMethodsViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                VStack(spacing: 0) {
+                    // Header
+                    HStack(spacing: Layout.SecureHeader.hSpacing) {
+                        Image(systemName: "checkmark.shield")
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: scale * Layout.SecureHeader.iconWidth, height: scale * Layout.SecureHeader.iconHeight)
+                            .tint(Color(.accent))
+
+                        Text(Localization.transactionsSecure)
+                            .foregroundColor(Color(.text))
+                            .subheadlineStyle()
+
+                        Spacer()
+                    }
+                    .padding(.horizontal, Layout.SecureHeader.hPadding)
+                    .padding(.vertical, Layout.SecureHeader.vPadding)
+                    .background(Color(.systemBackground))
+
+                    // Payment Methods list
+                    ListHeaderView(text: Localization.paymentMethodsHeader, alignment: .left)
+                        .textCase(.uppercase)
+                        .renderedIf(viewModel.paymentMethods.isNotEmpty)
+
+                    // Empty state when there are no payments methods
+                    VStack(alignment: .center) {
+                        Spacer()
+                            .frame(height: Layout.spacerHeight)
+                        EmptyState(title: Localization.pleaseAddPaymentMethodMessage, image: .waitingForCustomersImage)
+                    }
+                    .renderedIf(viewModel.paymentMethods.isEmpty)
+
+                    Divider()
+                        .renderedIf(viewModel.paymentMethods.isNotEmpty)
+
+                    ForEach(viewModel.paymentMethods) { method in
+                        let selected: Bool = {
+                            guard let selectedPaymentMethodID = viewModel.selectedPaymentMethodID else {
+                                return false
+                            }
+                            return method.id == selectedPaymentMethodID
+                        }()
+
+                        SelectableItemRow(title: "\(method.type.rawValue.capitalized) ****\(method.info.lastDigits)",
+                                          subtitle: method.name,
```

Comment:
According to the design, we should display the card type and cardholder name, not the method type and name.

```suggestion
                        SelectableItemRow(title: "\(method.info.type) ****\(method.info.lastDigits)",
                                          subtitle: method.info.cardholderName,
```

Reply:
Changed this as part of the refactor to list work in 2c4e6ba1576f3db9cd1ab5a5370be1113acd1637